storybookjs/storybook
# Migrate @storybook/html to Typescript

Issue: #5030

## What I did

Migrate @storybook/html to Typescript

## How to test

Everything should function the same, except the package should now have type definitions.

_Note:_ The first commit is intentionally mostly a rename from js => ts to make the review easier via the subsequent commits.
